news/blog/_posts/2021-04-19-cadence-ui-3-25-0.md

```diff
@@ -1,5 +1,6 @@
 ---
-date: 2021-04-19T07:00:01
+date: 2021-04-19T07:00:03
+image: https://user-images.githubusercontent.com/58960161/114792371-1c66d800-9d3d-11eb-8132-f807c7c83d98.png
```

Comment:
> pending task image to latest UI release

Reply:
Is it possible to add one introductory sentence before the screenshot? Something like "Now you can view all your pending activites/childworkflows in the pending tasks tab"

Reply:
yes you can add a optional summary. I will update with the summary text and update the screenshot.

Reply:
PR updated @meiliang86